### DISCUSSION TITLE
자산의 변동사항을 한번에 확인할 수 있는 앱

### DISCUSSION THREAD

**Discussion by ypd01018** [ondal-house/zu-story#15] [아이디어 선정 안건]
- 달러, 금값, 주식, COIN 등
- 데이터 연관성을 볼 수 있지 않을까?
- 사용자 커스텀 대시보드 ?<- 약간 빌님 의견(4번)이랑도 일맥상통

Pros
- 유행이 조금 지나긴했지만 모두가 궁금했던 주제로 시작을 할 수 있을 것
- 자세히는 모르지만 주가 예측 프로그램에 대한 예시는 꽤나 있던 것으로 기억 (단순 수준이겠지만)

Cons 
- 우리가 데이터를 수집할 수 있는 범위가 단순 크롤링 밖에 생각이 나지 않음..
- 생각보다 데이터 수집이 힘들지도..

**Comment by gychoi0916** [comment]
1) 자산 or 자산성 지표 ? 
자산성 지표 
2) 화면에 그릴게 많아서 프론트 위주의 공수가 클 것으로 예상 UX 위주의 프로젝트가 될 가능성 -> 프론트로써는 흥미 
데이터 가져오는 방식이 블록커가 될 수 도 있을 것 
3) 어디까지 프론트의 범주로 둘 것인가 (차트의 다양성 등 ) , 제한을 두고 기능을 붙인다면? -> db 활용 , 커뮤니티 등 을 붙일 수도 있을 것 

실시간 업데이트 등의 어려움 //
